cmd/ponzu/add.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// use `go get` to download addon and add to $GOPATH/src, useful
+// for IDE auto-import and code completion, then copy entire directory
+// tree to project's ./addons folder
+func getAddon(args []string) error {
+
+	var cmdOptions []string
+	var addonPath = args[1]
+
+	// Go get
+	cmdOptions = append(cmdOptions, "get", addonPath)
+	get := exec.Command(gocmd, cmdOptions...)
```

Comment:
good catch to re-use the gocmd variable.. will be very useful when working with future go version betas / release candidates. thank you for being mindful about that!